frontend/components/account/pug/account-modelsetting.pug

```diff
@@ -40,18 +40,23 @@
 
 					div(layout="row" flex)
 						div(flex="50")
-							md-input-container.modelSettingInput(layout-fill layout-margin)
-								label Issue topic types
-								textarea(type="text", ng-model="vm.topicTypes")
-								.inputHint One line per topic type
+							md-chips(
+								ng-model="vm.topicTypes"
+								name="topicTypes"
+								md-removable="true"
+								md-transform-chip="vm.newTopicType($chip)"
```

Comment:
Here :)

Reply:
can we call it convertTopicType or something?

It's mildly confusion that somewhere down the line you have a variable called newTopicTypes. And at one point I was wondering why you are trying to pass a function around lol

Reply:
I'm not sure convertTopicType is a good name as it's not really converting anything. Maybe `createTopicType` ?

Reply:
whatever works, just anything but the singular of newTopicTypes.